tokio-console/src/view/tasks.rs

```diff
@@ -122,7 +122,7 @@ impl TableList<11> for TasksTable {
                         warnings,
                         Cell::from(id_width.update_str(format!(
                             "{:>width$}",
-                            task.id(),
+                            task.task_id().map(|id| id.to_string()).unwrap_or_default(),
```

Comment:
since the task ID is not going to change over the lifetime of the task, and we have to format an ID for each task every time we draw the task list, i wonder if we want to cache a string representation of the task's ID as soon as we receive a task --- we could have an `id_str` field on the `Task` struct, which would allow us to avoid calling `to_string()` on the ID for every task every time we render the tasks list?

Reply:
I need to check the docs, but do we really have to format all rows in a table every time it's opened? Do we not only format the visible rows?

Reply:
er, that's what i meant. the point is that this work is performed every time we redraw the table.

Reply:
OK, got it. I understand now.

Reply:
@hawkw  I've made some changes to store a string of the id. Please let me know if this is what you had in mind.